build/org.eclipse.birt.target/org.eclipse.birt.target.target

```diff
@@ -330,18 +338,18 @@
       <repository location="https://download.eclipse.org/justj/jres/17/updates/release/latest"/>
     </location>
     <location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
-      <unit id="biz.aQute.bndlib" version="0.0.0"/>
       <unit id="org.eclipse.e4.tools.emf.ui" version="0.0.0"/>
       <unit id="org.eclipse.e4.tools.services" version="0.0.0"/>
       <unit id="org.eclipse.ecf.core.feature.feature.group" version="0.0.0"/>
       <unit id="org.eclipse.ecf.core.ssl.feature.feature.group" version="0.0.0"/>
       <unit id="org.eclipse.ecf.filetransfer.feature.feature.group" version="0.0.0"/>
-      <unit id="org.eclipse.ecf.filetransfer.httpclient5.feature.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.ecf.filetransfer.httpclientjava.feature.feature.group" version="0.0.0"/>
       <unit id="org.eclipse.ecf.filetransfer.ssl.feature.feature.group" version="0.0.0"/>
       <unit id="org.eclipse.equinox.executable.feature.group" version="0.0.0"/>
       <unit id="org.eclipse.rcp.configuration.feature.group" version="0.0.0"/>
       <unit id="org.eclipse.sdk.feature.group" version="0.0.0"/>
-      <repository location="https://download.eclipse.org/eclipse/updates/4.29/R-4.29-202309031000"/>
+      <unit id="org.eclipse.search.core" version="0.0.0"/>
+      <repository location="https://download.eclipse.org/eclipse/updates/4.30-I-builds"/>
```

Comment:
@claesrosell i it possible to replace such short-living repositories with long-living, at least on releasing?
Now in BIRT release 4.14.0 [stands](https://github.com/eclipse-birt/birt/blob/96394b3b0043b5f06c23d0b43c076c5ef48cca47/build/org.eclipse.birt.target/org.eclipse.birt.target.target#L138) short-living [non-existent](https://download.eclipse.org/eclipse/updates/4.30-I-builds) eclipse platform. After replacing with [long-living](https://download.eclipse.org/eclipse/updates/4.30/) eclipse platform repository project does not assemble. Version 4.14.0 cannot be modified, also with minor changes.
The same thing can acquire with future release 4.15.0 -- now in master stands also [short-living](https://github.com/eclipse-birt/birt/blob/f14f2c9b3a5030a16aa79a0c735960d90c1d0546/build/org.eclipse.birt.target/org.eclipse.birt.target.target#L142) repository